eeraster.py

```diff
@@ -15,6 +15,8 @@
 _CRS = 'EPSG:3857'
 _demXfab = None
 _dem = None
+_BUCKET_NAME = "unprocessed-isoscapes"
+DEFAULT_OXYGEN_EE_ASSET = "projects/river-sky-386919/assets/isoscapes/d18O_isocape"
```

Comment:
this is our test environment and its probably the best place for the script to place the isoscapes and allows testing with our test TimberID environment.
We can keep the copy from test to production as a manual step.
But you should probably add a comment about this fact and that to enable the isoscape in TimberID production requires another step in EE codeeditor